x/launch/client/cli/query_request_selection.go

```diff
@@ -0,0 +1,52 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/tendermint/spn/x/launch/types"
+)
+
+var _ = strconv.Itoa(0)
```

Comment:
```suggestion
```

This is generated by Starport CLI for some utilities like not throwing errors because of unused packages
It's not a big deal to have these in the code but don't hesitate to remove them when you spot them